src/engine/strat_engine/filesystem.rs

```diff
@@ -86,6 +86,11 @@ impl StratFilesystem {
     pub fn thin_id(&self) -> ThinDevId {
         self.thin_dev.id()
     }
+
+    /// Tear down the filesystem.
```

Comment:
Is there more that this method should be doing, i.e., is there a TODO that belongs here?